chapters/stress/instructions.md

```diff
@@ -0,0 +1,38 @@
+### Instructions for replicating Jina cloud benchmark
+
+You can reproduce this benchmark yourself. You have to do the following:
+
+1. Configure and start the instances in AWS
+
+    You can try using another provider. Additionally, you can try running it in Docker Compose, on your local machine. However, the Annoy & Faiss indexers use their own Docker images and running a Docker within Docker is currently not supported.
+
+1. On all the machines, apart from the client: 
+   
+    1. Install Jina, JinaD and the other requirements on each of the machines
+
+        The `requirements.txt` files can be found in the folders `image` or `text`. Install the ones you plan to run.
+  
+1. Do the following in the client machine:
```

Comment:
Some explanation on what operations are run in the `run_test.sh`

Reply:
Like saying that first index for X time, then query with annoy, faiss and NumpyIndexer

Reply:
Done